docs/methodology.md

```diff
@@ -1,48 +1,52 @@
 # Methodology
+
 Below we describe the methodology used to benchmark [CLP] and other tools.
```

Comment:
We need more of an intro. It should cover:

* that we benchmark ingestion and query performance for each tool.
* that clp supports both unstructured and dynamically-structured logs, so we have separate benchmarks for each.
* that below, we describe the datasets, benchmark machines, execution flow, and metrics collected.